src/components/LogInorSignUp.vue

```diff
@@ -58,6 +71,12 @@ export default {
         this.dialog = false;
       }
     },
+    googleSignin() {
```

Comment:
this is not part of this ticket. please to move to the appropriate branch and clean this one up.